### ISSUE TITLE
1

### ISSUE THREAD

**Issue by niannian28** [akashyadav08/Diff_LSTM#1]
Hello, regarding the network model you proposed, I think the comparison model is too low-level. It is already 2025, and few people are still using RNN and CNN for time series prediction.

**Comment by akashyadav08** [comment]
Hey there, a broader question is not which models are better but a) can we do a sound quantitative comparison between them and b) if newer facets of this architecture can be applied to newer models. For both these questions, this base model of Diff-LSTM works well. One can make a rather complex model which performs well with larger training data or only for specific tasks (for which you will find abundance of research papers but what they lack is a baseline comparison). Question answered with Diff-LSTM has more to do with a more general observation, can data augmentation with modified loss functions do better in bias-variance tradeoff. This is very close to PINNs in this regard but argued for more general cases.
